drivers/gpio/gpio_ite_it8xxx2.c

```diff
@@ -0,0 +1,420 @@
+/*
+ * Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+#include <errno.h>
+#include <device.h>
+#include <drivers/gpio.h>
+#include <zephyr/types.h>
+#include <sys/util.h>
+#include <string.h>
+#include <logging/log.h>
+#include "gpio_utils.h"
+
+#define DT_DRV_COMPAT
+#define GPIO_LOW        0
+#define GPIO_HIGH       1
+#define NUM_IO_MAX              8
+#define LOG_LEVEL CONFIG_GPIO_LOG_LEVEL
+LOG_MODULE_REGISTER(gpio_ite);
+
+extern void ite_intc_irq_enable(unsigned int irq);
+extern void ite_intc_irq_disable(unsigned int irq);
+
+#define GPIO_GPDRB              (EC_Register_Base_Address + 0x1602)
+#define GPIO_GPDRF              (EC_Register_Base_Address + 0x1606)
+#define GPIO_GPDRM              (EC_Register_Base_Address + 0x160d)
+#define GPCR_OFFSET             (0x10)
+
+struct gpio_ite_wui {
+	volatile uint8_t *reg_addr;
+	volatile uint8_t *clear_addr;
+	volatile uint8_t *bothedge_addr;
+	uint32_t pin;
+	uint32_t irq;
+};
+static const struct gpio_ite_wui GPIO_GPDRB_wui[] = {
```

Comment:
blank lines after }; in each case

What is WUEMR10? A macro?

Reply:
it was register, I've added comment about it.